Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2022 MobileCoin Inc.
-FROM alpine/helm:3.9.3
+FROM alpine:edge
```

Comment:
I think this makes sense;  the bleeding edge might give us headaches at some point.

Reply:
ends up that `alpine/` isn't alpine official,  just some random org that may or may not be related.

Reply:
Edge because kubectl and helm are in the edge/testing channels.